keps/sig-storage/5489-default-storage-class-per-access-mode/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Default StorageClass per Volume Access Mode
+kep-number: 5489
+authors:
+  - "@savirg"
+owning-sig: sig-storage
+participating-sigs:
+status: implementable
+creation-date: 2025-08-22
+reviewers:
+  - "@msau42"
+approvers:
+  - "@msau42"
+
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+# If the purpose of this KEP is to deprecate a user-visible feature
+# and a Deprecated feature gates are added, they should be deprecated|disabled|removed.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.35"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.35"
```

Comment:
Should this be beta? Or we still go through 3 release cycles?

Reply:
alpha, beta, GA is the typical cycle